src/Game/UI/uiPauseMenuDataMgr.h

```diff
@@ -366,6 +367,9 @@ class PauseMenuDataMgr {
     void equipWeapon(PouchItem* weapon);
     void unequip(PouchItem* item);
 
+    // FIXME: types
+    int x_28(void* unk, int a3, PouchItem* item);
```

Comment:
```suggestion
    bool useItemFromRecipeAndSave(void* recipe, int multiplier, PouchItem* item);
```
avoid placeholder names as much as you can